pkg/kn/traffic/compute_test.go

```diff
@@ -278,6 +278,18 @@ func TestComputeErrMsg(t *testing.T) {
 			[]string{"--traffic", "echo-v1=40", "--traffic", "echo-v1=60"},
 			"repetition of revision reference echo-v1 is not allowed, use only once with --traffic flag",
 		},
+		{
+			"untag single tag that does not exist",
+			append(newServiceTraffic([]servingv1.TrafficTarget{}), newTarget("latest", "echo-v1", 100, false)),
+			[]string{"--untag", "foo"},
+			"Error: tag foo does not exist",
```

Comment:
See my first comment: I would never check on the exact wording of an error message as this might change (e.g. like for the suggestions I made, e.g. to avoid the `Error` prefix and add the service information.

Reply:
Due to the structure of the tests, can we leave this in place for now? They seem to focus on the exact error message via `assert.Error(t, err, testCase.errMsg)`. However, would be happy to adapt the tests here maybe as a separate issue/pr.

Reply:
I'm fine with that.